addon/components/pix-select.hbs

```diff
@@ -66,7 +66,7 @@
             ' pix-select-options-category__option--selected'
           }}{{unless this.displayDefaultOption ' pix-select-options-category__option--hidden'}}"
         role="option"
-        tabindex={{if this.isExpanded "0" "-1"}}
```

Comment:
Il faut garder la condition `isExpanded`. (sinon elle sera focus quand la liste sera caché).

`get tabIndexValue() { return this.isExpanded && ! this.args.hideDefaultOption ? "0" : "-1"; }` ?

Reply:
Ça marche déjà avec l'option par défaut visible https://ui-pr340.review.pix.fr/?path=/story/form-select--default

Reply:
Exact, je corrige !